api/api.go

```diff
@@ -1562,28 +1561,10 @@ func (api *Server) estimateActionGasConsumptionForExecution(exec *iotextypes.Exe
 		}
 		return nil, status.Error(codes.Internal, "execution simulation is failed")
 	}
-	estimatedGas := receipt.GasConsumed
-	enough, err := api.isGasLimitEnough(callerAddr, sc, nonce, estimatedGas)
-	if err != nil {
-		return nil, status.Error(codes.Internal, err.Error())
-	}
-	if !enough {
-		low, high := estimatedGas, api.cfg.Genesis.BlockGasLimit
-		estimatedGas = high
-		for low <= high {
-			mid := (low + high) / 2
-			enough, err = api.isGasLimitEnough(callerAddr, sc, nonce, mid)
-			if err != nil {
-				return nil, status.Error(codes.Internal, err.Error())
-			}
-			if enough {
-				estimatedGas = mid
-				break
-			}
-			low = mid + 1
-		}
-	}
 
+	// for our EVM, 2*GasConsumed will make it consume all the input gas
+	// add 25% as safety margin
+	estimatedGas := 2*receipt.GasConsumed + (receipt.GasConsumed >> 1)
```

Comment:
This is 2.5x, while the comment is 2.25x

Reply:
exact amount = 2 * gasConsumed
add 25% = 2 * gasConsumed * 1.25 = 2.5 * gasConsumed